IndomitableIbises/Bangazon-Sprint-2
# initial commit | created django app and setup file structure

# Changes Made
1. initialized the django app
1. setup the file structure for models, tests, views, and templates

## Steps to Test
1. ```git fetch --all``` & ```git checkout startproject```
1. Ensure there are no issues with the project or file structure
